design/pipeline-artifacts.md

```diff
@@ -33,20 +33,19 @@ Artifacts are distinct from other `resources` types, including `containers`, `re
 
 ---
 
-## Downloading artifacts: `downloadArtifact`
+## Downloading artifacts: `download`
 
-`downloadArtifact` is a shortcut for several tasks:
+`download` is a shortcut for two tasks:
 
 - The [Download Pipeline Artifacts](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/download-pipeline-artifact) task
 - The [Download Fileshare Artifacts](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/download-fileshare-artifacts) task
-- Download tasks for other pipelines systems e.g. Jenkins
 
-It will download artifacts published from a previous job or stage or from another pipeline. Artifacts are downloaded either to `$PIPELINES_RESOURCESDIR` or to the directory specified in `root`.
+It will download artifacts uploaded from a previous job or stage or from another pipeline. Artifacts are downloaded either to `$PIPELINES_RESOURCESDIR` or to the directory specified in `root`.
```

Comment:
What is `$PIPELINES_RESOURCESDIR`? Is this a new folder under the build directory? What is the folder name? `r`?

Have you considered the singular form? `PIPELINE_RESOURCESDIR`?

All other directory varibles spell out the full word "DIRECTORY" (except for `agent.workFolder`). Shouldn't we be consistent here and name the variable `pipeline.resourcesDirectory`?

Reply:
What if you specify a relative path? Would it be rooted against the System.DefaultWorkingDirectory (i.e. Build.SourcesDirectory in build)? That is how filePath inputs work today. Or is this a regular string input?

Reply:
I'd pulled those names from Matt's [branch on variables](https://github.com/Microsoft/azure-pipelines-yaml/blob/proposals/variables/design/variables-in-pipelines.md). I don't want to define a new directory here - my intent is to use the existing directory. I'm also happy to align on variable names. So $(Pipeline.ResourcesDirectory) or $PIPELINE_RESOURCESDIRECTORY then?

Reply:
Sorry Eric, I caused the confusion here. We need to get aligned on variables, and whatever we align behind is what artifacts will use. Let me continue that conversation in the other PR.

Reply:
Great. I'll close this conversation for now.